src/library/Uno.Material/Controls/Chips/Chip.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls.Primitives;
```

Comment:
Should we have a ICommand property on Chip? For the ActionChip

Reply:
`Chip` already offers it (via its `ButtonBase` ancestry).